src/wrap.c

```diff
@@ -0,0 +1,44 @@
+#include <curses.h>
+
+#ifndef NCURSES_VERSION
+/*
+ * Some things are declared differently in NetBSD Curses; these are handled here
+ */
+
+int (COLOR_PAIR)(int n) {
+	return COLOR_PAIR(n);
+}
+
+bool (has_mouse)(void) {
+	return has_mouse();
+}
+
+int (getmouse)(MEVENT *event) {
+	return getmouse(event);
+}
+
+int (ungetmouse)(MEVENT *event) {
+	return ungetmouse(event);
+}
+
+mmask_t (mousemask)(mmask_t newmask, mmask_t *oldmask) {
+	return mousemask(newmask, oldmask);
+}
+
+bool (wenclose)(const WINDOW *win, int y, int x) {
+	return wenclose(win, y, x);
+}
+
+bool (mouse_trafo)(int *pY, int *pX, bool to_screen) {
+	return mouse_trafo(pY, pX, to_screen);
+}
+
+bool (wmouse_trafo)(const WINDOW *win, int *pY, int *pX, bool to_screen) {
+	return wmouse_trafo(win, pY, pX, to_screen);
+}
+
+int (mouseinterval)(int erval) {
+	return mouseinterval(erval);
+}
```

Comment:
due to [this commit](https://github.com/sabotage-linux/netbsd-curses/commit/62e762344efd9666725ccf74b8847aef7c862b43) and [this commit](https://github.com/NetBSD/src/blob/trunk/lib/libcurses/mouse.c), these will have to be removed before merging